backend/resources/migrations/postgres/20250814121300_requires_reauthentication.up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE oidc_clients ADD COLUMN requires_reauthentication BOOLEAN NOT NULL DEFAULT FALSE;
+
+CREATE TABLE reauthentication_tokens (
+    id TEXT PRIMARY KEY,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Do we need to store the time zone? The common practice is to always store times in the DB in UTC, and then format in the appropriate time zone when displaying